packages/story-editor/src/components/footer/layers/layers.js

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useState } from '@googleforcreators/react';
+import { __ } from '@googleforcreators/i18n';
+import {
+  Icons,
+  Text,
+  NotificationBubble,
+  THEME_CONSTANTS,
+  BUBBLE_VARIANTS,
+} from '@googleforcreators/design-system';
+
+/**
+ * Internal dependencies
+ */
+import styled, { css } from 'styled-components';
+import { LayerPanel } from '../../panels/design';
+import useLayers from '../../panels/design/layer/useLayers';
+import Popup from '../../secondaryPopup';
+
+const Container = styled.div`
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  width: 300px;
+  border-radius: ${({ theme }) => theme.borders.radius.small};
+  background-color: ${({ theme }) => theme.colors.bg.secondary};
+  overflow: auto;
+`;
+
+export const LabelText = styled(Text)`
+  color: ${({ theme }) => theme.colors.fg.primary};
+  padding-right: 8px;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: hidden;
+`;
+
+const Button = styled.button`
```

Comment:
Hm, we should really use a button from the design-system here -- whenever there's a button without variant in Figma, would be good to confirm with UX to check which button it is. The only thing that I can see in Figma right now is that it's a rectangle button of size small, unfortunately variant hasn't been defined.

Otherwise we'd need to also add hover/focus etc. here separately.
From [Figma](https://www.figma.com/file/Ha6kk9yXDN1rB80NzcDf2J/_Editor---Design-System-%22Style-Guide%22?node-id=3821%3A81609), it looks like it's a quaternary button, let's use that instead. Then we also wouldn't probably need to style the `LabelText` separately.

Reply:
So this is actually more in line with the other popups in the footer: 
<img width="334" alt="Screen Shot 2022-03-10 at 10 53 16 AM" src="https://user-images.githubusercontent.com/10720454/157725071-8677f8e9-9e50-49f3-a034-1905696b977b.png">

Reply:
The only reason the caret is on the zoom is because it's a dropdown(up) menu. I was able to reuse the `ToggleButton` that the rest of the footer popups utilize with only a few modifications. I'll screenshot the diff here. 

I'd like to request you just leave the z index be and not further tinker with it. It's been really hard to untangle and my PR will resolve your issues with it. 
<img width="550" alt="Screen Shot 2022-03-10 at 10 55 36 AM" src="https://user-images.githubusercontent.com/10720454/157725399-d5fd4954-279b-4320-8bf3-1f2a91f45664.png">
<img width="596" alt="Screen Shot 2022-03-10 at 10 55 25 AM" src="https://user-images.githubusercontent.com/10720454/157725401-1f808f39-b99b-4d29-ac08-80b4d93190c3.png">

Reply:
Amy might be fine with the design change to cater to consistency. 

That said, if one notification bubble has a bg and a circle then they _all_ should and that could be it's own lower priority ticket.

Reply:
Just pasting the last input from Amy:

<img width="934" alt="image" src="https://user-images.githubusercontent.com/21036224/157937259-add08316-2275-4362-84a8-a9834a260c31.png">

(so Q-styled button with up/down arrow).
But now since you made the change maybe just double-check with her.

Reply:
Chatted with amy, thanks for the screenshot of your chat it's helpful (slack won't open the links). 

we're going to handle the chevron/updates from the existing toggle button separate to unblock work and be consistent with other buttons with same behavior. (#10891 )